recipes/cunit/all/conanfile.py

```diff
@@ -101,23 +113,23 @@ def _configure_autools(self):
             conf_args.extend(["--enable-shared", "--disable-static"])
         else:
             conf_args.extend(["--disable-shared", "--enable-static"])
-        self._autotools.configure(args=conf_args)
+        self._autotools.configure(args=conf_args, host=host, build=build)
         return self._autotools
 
     def build(self):
         for patch in self.conan_data.get("patches", {}).get(self.version, []):
             tools.patch(**patch)
         with self._build_context():
             with tools.chdir(self._source_subfolder):
-                self.run("autoreconf -fiv".format(os.environ["AUTORECONF"]), win_bash=tools.os_info.is_windows)
-                autotools = self._configure_autools()
+                self.run("{} -fiv".format(tools.get_env("AUTORECONF")), win_bash=tools.os_info.is_windows)
+                autotools = self._configure_autotools()
                 autotools.make()
 
     def package(self):
         self.copy("COPYING", src=self._source_subfolder, dst="licenses")
         with self._build_context():
             with tools.chdir(self._source_subfolder):
-                autotools = self._configure_autools()
+                autotools = self._configure_autotools()
                 autotools.install()
 
         os.unlink(os.path.join(self.package_folder, "lib", "libcunit.la"))
```

Comment:
```suggestion
                autotools.install()

        if self.settings.compiler == "Visual Studio" and self.options.shared:
            tools.rename(os.path.join(self.package_folder, "lib", "libcunit.dll.lib"),
                         os.path.join(self.package_folder, "lib", "libcunit.lib"))

        os.unlink(os.path.join(self.package_folder, "lib", "libcunit.la"))
```

Reply:
And in `package_info()`, replace:

```python
        libname = "cunit"
        if self.settings.os == "Windows" and self.options.shared:
            libname += ".dll" + (".lib" if self.settings.compiler == "Visual Studio" else ".a")
        self.cpp_info.libs = [libname]
```

by

```python
        self.cpp_info.libs = ["cunit"]
```